README.md

```diff
@@ -52,20 +53,25 @@ package main
 
 import (
 	"fmt"
+	"time"
 
 	"github.com/curvegrid/gofig"
 )
 
 type Config struct {
-	Debug       bool   `flag:"d" desc:"enable debugging"`
-	Environment string `json:"env" toml:"env" yaml:"env" env:"env" flag:"e" desc:"environment name"`
-	Port        int    `desc:"port to listen on"`
+	Debug       bool           `flag:"d" desc:"enable debugging"`
+	Environment string         `json:"env" toml:"env" yaml:"env" env:"env" flag:"e" desc:"environment name"`
+	Port        int            `desc:"port to listen on"`
+	Timeout     gofig.Duration `desc:"server timeout"`
 }
 
 func main() {
 	cfg := Config{}
 	cfg.Port = 5243 // user-defined default value
 
+	timeout, _ := time.ParseDuration("30s")
+	cfg.Timeout = gofig.Duration(timeout)  // gofig.Duration is time.Duration with added interfaces
```

Comment:
would `gofig.Duration(30 * time.Second)` make more sense as an example? Also not sure about the wording in the comment.